### PR TITLE
Domain Transfers: Updated first and second input placeholder text and colors

### PR DESCRIPTION
**Pull request by paulopmt1**
Related to 3173-gh-Automattic/dotcom-forge

## Proposed Changes

* Updated placeholder text
* Updated placeholder colors

image

## Testing Instructions

* Open [the domain transfer](http://calypso.localhost:3000/setup/domain-transfer/domains) and ensure it matches Figma (available at related card)

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx [hunk 1]
```diff
@@ -177,7 +177,7 @@ export function DomainCodePair( {
 							id={ id }
 							value={ domain }
 							className="domains__domain-name-input-field"
-							placeholder={ __( 'Please enter the domain name and authorization code.' ) }
+							placeholder={ __( 'example.com' ) }
```
Comment (allilevine) [inline client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx:180]: Does example.com need a translation? 🤔

Comment (paulopmt1) [reply client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx:180]: Yeah, I was thinking the same, but [after checking we currently have it](https://translate.wordpress.com/projects/wpcom/-all-translated/233557/) and in some languages it looks way different from EN, it seems to make sense offering it.

**Review by allilevine — APPROVED** [review]
This addresses the design feedback ✅ 
Screen Shot 2023-07-24 at 2 13 49 PM

File: client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx (hunk 1 above)
Comment (allilevine) [reply client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/domain-code-pair.tsx:180]: Interesting, thanks for checking!

**Pull request merged**